definitions/pipelines/rnaseq_star_fusion.cwl

```diff
@@ -192,3 +192,16 @@ steps:
             bam: mark_dup/sorted_bam
         out:
             [metrics, chart]
+    bam_to_cram:
+        run: ../tools/bam_to_cram.cwl
+        in:
+          reference: cdna_fasta
```

Comment:
The cdna fasta is not the same as the reference sequence fasta.  It contains one fasta entry for each transcript.  The reference fasta is needed here, either derived from the star directory, or passed in explicitly

Reply:
Mybad!